core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -99,6 +99,7 @@ $(call force,CFG_DRIVERS_REGULATOR,y)
 $(call force,CFG_GIC,y)
 $(call force,CFG_INIT_CNTVOFF,y)
 $(call force,CFG_PSCI_ARM32,y)
+$(call force,CFG_REGULATOR_GPIO,y)
```

Comment:
It looks like `force` is being abused here, because I suppose the platform did work before this commit, which implies that `CFG_REGULATOR_GPIO` disabled is a valid config.

Reply:
Indeed the platform does not mandate `CFG_REGULATOR_GPIO`. I think it's simpler embeding the driver and let OP-TEE DT tell whether it's used or not.

I can switch to `CFG_REGULATOR_GPIO ?= y`. @GseoC, what do you think?

Reply:
comment addressed.